File: week1/me/hello.c
#include <cs50.h>
#include <stdio.h>

// This program takes the name of the user and returns a greeting
int main(void)
{
    // 'answer' is the user's input and could be substituted for a string return value from a method
    string answer = get_string("What's your name?");
    printf("Hello, %s\n", answer);
}

File: week6/sentimental-hello/hello.py
import cs50

# create w while loop that repeats as long as the name entered by user is blank
while True:
    # prompt user for a string (their name)
    name = cs50.get_string("What is your name? ")
    if name != "":
        # name is not blank - break out of loop
        break

# print the customized greeting with the user's name
print(f"hello, {name}")
